docker/run-build

```diff
@@ -35,11 +35,11 @@ docker run --rm \
 
     mkdir  /tmp/results
     cp -r /tmp/source_downloads /tmp/results/source_downloads
+    sudo chown mysql:mysql /tmp/ps/mysql-test/collections /tmp/ps/storage/rocksdb/rocksdb/util
```

Comment:
this line needed because we modify source tree during cmake/make:
- https://jira.percona.com/browse/PS-4536
- https://jira.percona.com/browse/PS-4537

Reply:
This line is going to stay even if the above are fixed due to need to build upstream server and old versions